dashbuilder-backend/dashbuilder-dataset-sql/src/test/java/org/dashbuilder/dataprovider/sql/SQLDataSetTrimTest.java

```diff
@@ -31,4 +42,41 @@ public void testTrim() throws Exception {
         subTest.testTrim();
         subTest.testDuplicatedColumns();
     }
+
+    @Test
+    public void testAvoidDuplicatedGroupColumn() throws Exception {
+
```

Comment:
Please remove unnecessary empty lines at the beginning and at the end of the method.